apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -1042,4 +1054,18 @@ export class ApiModelGenerator {
     }
     return parameters;
   }
+
+  private _determineReadonly(astDeclaration: AstDeclaration): boolean {
```

Comment:
If this `isReadonly` flag only applies to properties, property signatures, and variables, does it make sense to inline this logic within each specific "process" method instead of pulling it out into a helper? I don't feel super strongly either way.

Reply:
I'm slightly partial towards a helper since the TSDoc comment and getter logic would apply across the board, but I can separate out if greatly desired.